### PR TITLE
Moved peep release balloon code into a function

### PR DESCRIPTION
**Pull request by RMcTn**

### CODE REVIEW

File: src/openrct2/peep/peep.h [hunk 1]
```diff
@@ -786,4 +786,6 @@ void increment_guests_heading_for_park();
 void decrement_guests_in_park();
 void decrement_guests_heading_for_park();
 
+void release_balloon(rct_peep *peep);
```
Comment (janisozaur) [inline src/openrct2/peep/peep.h]: Why not `static` in peep.c?

File: src/openrct2/peep/peep.c [hunk 2]
```diff
@@ -12932,3 +12917,17 @@ void decrement_guests_heading_for_park(){
         log_error("Attempt to decrement guests heading for park below zero.");
     }
 }
+
+void release_balloon(rct_peep *peep) {
```
Comment (Gymnasiast) [inline src/openrct2/peep/peep.c]: peep_release_balloon would be more consistent with other function names.

Also, if this function is not used outside peep.c, you can declare it static.

File: src/openrct2/peep/peep.c [hunk 3]
```diff
@@ -12932,3 +12918,17 @@ void decrement_guests_heading_for_park(){
...
 }
+
+static void peep_release_balloon(rct_peep *peep) {
+    if (peep->item_standard_flags & PEEP_ITEM_BALLOON)
+    {
+        peep->item_standard_flags &= ~PEEP_ITEM_BALLOON;
+
+        if (peep->sprite_type == PEEP_SPRITE_TYPE_BALLOON && peep->x != MAP_LOCATION_NULL)
+        {
+            create_balloon(peep->x, peep->y, peep->z + 10, peep->balloon_colour, false);
```
Comment (Gymnasiast) [inline src/openrct2/peep/peep.c]: The height appears to differ from both functions. I haven't checked by much, though. If it's insignificant it's ok, but I want to check first.

Comment (RMcTn) [inline src/openrct2/peep/peep.c]: I tested it in game and the code that uses 'height' seems to spawn a balloon a bit closer to the water. The function could take a height parameter though

Comment (Gymnasiast) [inline src/openrct2/peep/peep.c]: That might be better, yes.

Comment (janisozaur) [inline src/openrct2/peep/peep.c]: That's very good spot @Gymnasiast! If it's not a parameter that is kept, network version change will be required.

**Comment by RMcTn** [comment]
Made sure to use the same z positions that were originally used for spawning balloons

**Pull request merged**